<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectInvervalDailog</class>
 <widget class="QDialog" name="SelectInvervalDailog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidgetStart">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidgetEnd">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>20</width>
     <height>301</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="BttnSet">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>661</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Виберіть дату</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
